examples/javaee/README.md

```diff
@@ -0,0 +1,156 @@
+```
+
+If multiple services are running, then it can be narrowed by specifying labels:
+
+```sh
+kubectl get -w po -l context=docker-k8s-lab,name=mysql-pod
+NAME        READY     STATUS    RESTARTS   AGE
+mysql-pod   1/1       Running   0          4m
+```
+
+This is also the selector label used by service to target pods.
+
+When a Service is run on a node, the kubelet adds a set of environment variables for each active Service. It supports both Docker links compatible variables and simpler `{SVCNAME}_SERVICE_HOST` and `{SVCNAME}_SERVICE_PORT` variables, where the Service name is upper-cased and dashes are converted to underscores.
+
+Our service name is ``mysql-service'' and so ``MYSQL_SERVICE_SERVICE_HOST'' and ``MYSQL_SERVICE_SERVICE_PORT'' variables are available to other pods. This host and port variables are then used to create the JDBC resource in WildFly.
+
+### Start WildFly Replication Controller
+
+WildFly is a lightweight Java EE 7 compliant application server. It is wrapped in a Replication Controller and used as the Java EE runtime.
+
+In Kubernetes a [_Replication Controller_](../../docs/user-guide/replication-controller.md) is responsible for replicating sets of identical pods. Like a _Service_ it has a selector query which identifies the members of it's set.  Unlike a service it also has a desired number of replicas, and it will create or delete pods to ensure that the number of pods matches up with it's desired state.
+
+Here is definition of the MySQL service: [wildfly-rc.yaml](wildfly-rc.yaml). Create this controller:
+
+```sh
+kubectl create -f examples/javaee/wildfly-rc.yaml
+```
+
+Check status of the pod inside replication controller:
+
+```sh
+kubectl get po
+NAME               READY     STATUS    RESTARTS   AGE
+mysql-pod          1/1       Running   0          1h
+wildfly-rc-w2kk5   1/1       Running   0          6m
+```
+
+### Access the application
+
+Get IP address of the pod:
+
+```sh
+kubectl get -o template po wildfly-rc-w2kk5 --template={{.status.podIP}}
+10.246.1.23
+```
+
+Log in to minion and access the application:
+
+```sh
+vagrant ssh minion-1
+Last login: Thu Jul 16 00:24:36 2015 from 10.0.2.2
+[vagrant@kubernetes-minion-1 ~]$ curl http://10.246.1.23:8080/employees/resources/employees/
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?><collection><employee><id>1</id><name>Penny</name></employee><employee><id>2</id><name>Sheldon</name></employee><employee><id>3</id><name>Amy</name></employee><employee><id>4</id><name>Leonard</name></employee><employee><id>5</id><name>Bernadette</name></employee><employee><id>6</id><name>Raj</name></employee><employee><id>7</id><name>Howard</name></employee><employee><id>8</id><name>Priya</name></employee></collection>
+```
+
+### Delete resources
+
+All resources created in this application can be deleted:
+
+```sh
+kubectl delete -f examples/redis/mysql-pod.yaml
+kubectl delete -f examples/redis/mysql-service.yaml
+kubectl delete -f examples/redis/wildfly-rc.yaml
```

Comment:
Build is failing.

KUBE_ROOT}/hack/after-build/verify-generated-docs.sh" "$@"' exited with status 1
Call stack:
  1: ./hack/verify-generated-docs.sh:28 main(...)
Exiting with status 1
The command "./hack/verify-generated-docs.sh" exited with 1.
```

Should these s/redis/javaee/?